tests/integration/commands/dev/dev-miscellaneous.test.ts

```diff
@@ -1149,12 +1189,12 @@ describe.concurrent('commands/dev-miscellaneous', () => {
         })
         .withEdgeFunction({
           handler: () => {
-            // eslint-disable-next-line no-undef
-            const fromDenoGlobal = Deno.env.toObject()
-            // eslint-disable-next-line no-undef
+            // @ts-expect-error: We can't import Deno types without polluting the global environment
+            // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment, @typescript-eslint/no-unsafe-call, @typescript-eslint/no-unsafe-member-access
```

Comment:
It looks like `withEdgeFunction`'s `handler` arg also accepts a string, so another option would be to just pass this handler fixture as a raw string instead of pretending it's valid JS in this context.

Reply:
Yeah, I think we should do this for all the fixtures--I'll do it in a separate PR when I get back. (Another reason to do it: you can currently reference in-scope variables but the test will end up failing.)

Most editors (and Prettier) will recognize code in `javascript` and `typescript` template tags and format the code accordingly, so I think that's a better alternative.